client/pom.xml

```diff
@@ -222,5 +222,12 @@
             <artifactId>nebula-utils</artifactId>
             <version>${nebula-utils.version}</version>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.apache.commons/commons-pool2 -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-pool2</artifactId>
+            <version>2.6.2</version>
```

Comment:
move the version to `properties` and remove the comment

Reply:
done